modules/nvidia_plugin/src/cuda/graph.cpp

```diff
@@ -43,7 +43,7 @@ Handle(cudaGraphInstantiate, cudaGraphExecDestroy, g.get(), static_cast<cudaGrap
 }
 #if !defined(NDEBUG) || defined(_DEBUG)
 catch (std::exception &e) {
-    throw InferenceEngine::GeneralError { fmt::format("{}: {}", e.what(), errorMsg_) };
+    OPENVINO_THROW(fmt::format("{}: {}", e.what(), errorMsg_));
 }
 #endif
 // clang-format on
```

Comment:
First of all we don't disable clang-format before...
At the second, looks like clang format is disabled for this plugin. Can we enable it?

Reply:
It was just an idea

Reply:
> It was just an idea. But I propose to remove the redundant comment

Redundant comment was removed